api-client/src/main/java/com/xing/api/data/messages/Conversation.java

```diff
@@ -26,135 +29,43 @@
 /**
  * Java representation of a Conversation.
  */
-public class Conversation implements Serializable {
+@AutoValue
+public abstract class Conversation implements Serializable {
```

Comment:
try to write a test where one field is not present or null. It should crash with an NPE since AutoValues adds validations in the constructer of the generated class.